### PR TITLE
NET-288 kill daemon on uninstall

### PR DESCRIPTION
**Pull request by TobiaszCudnik**
## Describe your changes

NET-288 kill daemon on uninstall

## Provide testing steps

https://toil.kitemaker.co/cQARgD-Gravitl/6a50SD-Netmaker/items/288

## Checklist before requesting a review
- [ ] My changes affect only 10 files or less.
- [ ] I have performed a self-review of my code and tested it.
- [ ] If it is a new feature, I have added thorough tests, my code is <= 1450 lines.
- [ ] If it is a bugfix, my code is <= 200 lines.
- [ ] My functions are <= 80 lines.
- [ ] I have had my code reviewed by a peer.
- [ ] My unit tests pass locally.
- [ ] Netclient & Netmaker are awesome.

### CODE REVIEW

**Review by mattkasun — CHANGES_REQUESTED** [review]
what about changes to common_darwin, common_freebsd, common_windows?

File: daemon/common_unix.go [hunk 1]
```diff
@@ -21,8 +26,9 @@ func restart() error {
 	if err != nil {
 		return fmt.Errorf("failed to find running process for pid %d -- %w", pid, err)
 	}
-	if err := p.Signal(syscall.SIGHUP); err != nil {
-		return fmt.Errorf("SIGHUP failed -- %w", err)
+	slog.Info("Sending signal", s, "to PID", p)
```
[the remark this answers is not in the document]
Comment (TobiaszCudnik) [reply daemon/common_unix.go:29]: Docs: https://pkg.go.dev/golang.org/x/exp/slog
First example: `slog.Info("hello", "count", 3)`

**Review by afeiszli — APPROVED** [review]

**Review by Aceix — APPROVED** [review]

**Review by mattkasun — CHANGES_REQUESTED** [review]
slog call has incorrect number of args

File: daemon/common_unix.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (TobiaszCudnik) [reply daemon/common_unix.go]: Fixed.

**Review by mattkasun — APPROVED** [review]

**Review by mattkasun — APPROVED** [review]

**Review by abhishek9686 — APPROVED** [review]

**Review by theguy951357 — APPROVED** [review]
critical list passes

ps -aef | grep -i netclient shows no daemon running after uninstall command

**Pull request merged**